pandas/tests/groupby/test_categorical.py

```diff
@@ -525,13 +666,34 @@ def test_groupby_categorical_no_compress():
                        categories=["a", "b", "c", "d"], ordered=True)
     data = DataFrame({"a": [1, 1, 1, 2, 2, 2, 3, 4, 5], "b": cats})
 
-    result = data.groupby("b").mean()
+    result = data.groupby("b", observed=False).mean()
     result = result["a"].values
     exp = np.array([1, 2, 4, np.nan])
     tm.assert_numpy_array_equal(result, exp)
 
 
-def test_groupby_sort_categorical():
+def test_sort():
+
+    # http://stackoverflow.com/questions/23814368/sorting-pandas-categorical-labels-after-groupby
```

Comment:
Is this line too long? https://stackoverflow.com/q/23814368/1889400 is a short link to the same q.